src/Components/InfoContainer/OverviewContainer/OverviewContainer.js

```
@@ -0,0 +1,37 @@
+import HeaderMetadataContext from "../../../Providers/HeaderMetadataContext";
+
+import "./OverviewContainer.scss";
+
+/**
+ * @return {JSX.Element}
+ */
+export function OverviewContainer ({}) {
+    const overviewContainer = useRef();
+    const headerMetaContext = useContext(HeaderMetadataContext);
+
+    const [overview, setOverview] = useState(<></>);
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.

(lint/complexity/noUselessFragments)